### PR TITLE
[4.0] Namespace FinderIndexerResponse

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

Moves `FinderIndexerResponse` into its own file under `Joomla\Component\Finder\Administrator\Helper` namespace.

### Testing Instructions

Run Smart Search indexer.
Inspect the information in the modal.

### Expected result

Works like before.

### Documentation Changes Required

IDK.

### CODE REVIEW

File: libraries/extensions.classmap.php [hunk 1]
```diff
@@ -31,6 +31,7 @@
 JLoader::registerAlias('FinderIndexer',         '\\Joomla\\Component\\Finder\\Administrator\\Indexer\\Indexer', '5.0');
 JLoader::registerAlias('FinderIndexerParser',   '\\Joomla\\Component\\Finder\\Administrator\\Indexer\\Parser', '5.0');
 JLoader::registerAlias('FinderIndexerQuery',    '\\Joomla\\Component\\Finder\\Administrator\\Indexer\\Query', '5.0');
+JLoader::registerAlias('FinderIndexerResponse', '\\Joomla\\Component\\Finder\\Administrator\\Indexer\\Response', '5.0');
```
Comment (mbabker) [inline libraries/extensions.classmap.php]: You shouldn't need this one in the mapping array, this response class is internal to the component and not something intended for use outside of the one controller the code was present in.

File: administrator/components/com_finder/Indexer/Response.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_finder
+ *
+ * @copyright   Copyright (C) 2005 - 2020 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Finder\Administrator\Indexer;
```
Comment (mbabker) [inline administrator/components/com_finder/Indexer/Response.php]: I'd actually not put this class in the same namespace as the rest of the indexer.  The indexer namespace should be the code that's actually doing the indexing and should be considered library level API meant for extensions to build on and use, this class definitely doesn't fall in that category.

**Review by mbabker — CHANGES_REQUESTED** [review]

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 9bf102867cd629eab7ac87a4a674a595e4cb82ad<br /><br />Test successful. Deleted index and re-indexed content, created new content and re-indexed, and confirmed that statistics still pull data correctly.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27591">issues.joomla.org/tracker/joomla-cms/27591</a>.</sub>

File: libraries/extensions.classmap.php (hunk 1 above)
Comment (SharkyKZ) [reply libraries/extensions.classmap.php]: Removed.

File: administrator/components/com_finder/Indexer/Response.php (hunk 2 above)
Comment (SharkyKZ) [reply administrator/components/com_finder/Indexer/Response.php]: Moved to a helper class. Is that OK?

Comment (mbabker) [reply administrator/components/com_finder/Indexer/Response.php]: Better than being in the indexer namespace.  Truthfully if you are insistent upon putting it in a namespace it is better off in a `Response` namespace, not a `Helper` namespace (unless there is some mandate that the only namespaces a component can use are Controller, Model, Table, View, Helper, and maybe Form; the entire point of namespaces is to organize and structure your code, there should not be a mandate to throw classes in existing namespaces if none of the existing make sense).

Comment (SharkyKZ) [reply administrator/components/com_finder/Indexer/Response.php]: Moved to `Response` namespace.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 1bb0e235bebfdea86cda87e197f04a9af13246c9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27591">issues.joomla.org/tracker/joomla-cms/27591</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**